docs/apps/lnw/es2k/es2k-linux-networking.md

```diff
@@ -86,37 +83,37 @@ Packets coming from overlay network:
     ovs-vsctl add-port br-int <Physical port PR>
     ```
 
-##### Egress traffic with VxLAN encap
+##### Egress traffic with VxLAN encapsulation
 
 Packets coming from overlay network:
 
 - Determine the source port of the packet based on which overlay VSI the packet has landed on.
 - Validate if the source port is part of the bridge, else drop the packet.
-- If valid bridge configuration is found, find the PR associated with the bridge and forward the packet to the PR in ACC.
+- If valid bridge configuration is found, find the PR associated with the bridge and forward it to the PR in ACC.
 - OvS control plane receives the packet and forwards the packets to the destined VxLAN port if MAC is already learnt, else flood the packet in the respective bridge.
```

Comment:
I think using packets(plural) was a typo. 
We are talking about forwarding the packet here, so will use the term instead of using it.
Reworded for clarity